src/components/PriceRange.vue

```diff
@@ -69,7 +68,7 @@ export default {
     },
     from: {
       get() {
-        for (let refinement in this.searchStore.activeRefinements) {
+        for (const refinement in this.searchStore.activeRefinements) {
```

Comment:
just checked, works perfectly, we might just be two newbes not getting the spec here